config/routes.rb

```diff
@@ -23,7 +23,7 @@
   get "works/:id/file-other", to: "works#file_other", as: :work_file_other
   get "works/:id/review", to: "works#review", as: :work_review
   post "works/:id/review", to: "works#review"
-  post "works/:id/completed", to: "works#completed", as: :work_completed
+  post "works/:id/validate", to: "works#validate", as: :work_validate
```

Comment:
I changed the URL so that if the user looks at the URL in the browser (e.g. when there is an error during validation) they don't get the wrong impression that they have "completed" but see that we are still in the validation part of the process.